dev_guide/copy_files_to_container.adoc

```diff
@@ -40,6 +40,44 @@ $ oc rsync devpod1234:/src /home/user/source
 ----
 ====
 
+[[copying-usage-with-database-restore]]
+== Usage With Database Restore
+
+Use `oc rsync` to copy local database archives to a database pod's persistent volume directory.
+Note: MySQL is used here, but replace 'mysql|MYSQL' with 'pgsql|PGSQL' or 'mongodb|MONGODB' to use
```

Comment:
Let's place this note in a note box

[NOTE]
====
MySQL is used here, but replace 'mysql|MYSQL' with 'pgsql|PGSQL' or 'mongodb|MONGODB' to use any of the three supported databases. 
====